### ISSUE TITLE
How to return (string) name of effect?   (Feature request?) 

### ISSUE THREAD

**Issue by drscotthawley** [spotify/pedalboard#208]
Hey Peter.  How do we get the "name" of the effect? 

Currently running the code...
```python
x = Distortion()
print(x.__class__.__name__)
```
returns
```python
'pybind11_type'
```
rather than 'Distortion'. 

If there's currently no way to do this, then: Feature Request!  Either via `effect.__class__.__name__` or perhaps `effect.__name__` (my preference is the former)

**Comment by psobot** [comment]
Hey Scott!

That's a bit odd - with Pedalboard v0.7.1 running on Python 3.8, I get:
```python
In [1]: from pedalboard import Distortion

In [2]: x = Distortion()
   ...: print(x.__class__.__name__)
Distortion
```

Which versions of Python and Pedalboard are you running?

Here's an alternative hacky workaround for now, regardless:
```python
In [3]: repr(x)
Out[3]: '<pedalboard.Distortion drive_db=25 at 0x6000031ac140>'

In [4]: repr(x).split(".")[1].split(" ")[0]
Out[4]: 'Distortion'
```

**Comment by drscotthawley** [comment]
Oh! huh, yea on my Mac with Python 3.10.10 and  `pedalboard` 0.6.7 I _do_ get the proper class name. 
And in the Python REPL (from the shell) on the Ubuntu system from which I first saw the above-mentioned behavior, I get the proper name too. ??  That's Python 3.9.16 and `pedalboard` 0.7.0.  

But it was while running in Jupyter where I saw the problem. Let me try to reproduce what I was getting last night, using the same system:
![Screen Shot 2023-03-18 at 2 22 07 PM](https://user-images.githubusercontent.com/13925685/226132643-7e1e336e-68ed-4864-8ca9-6920bae2adbb.png)


???  Ok... Sorry to bother you. I don't know what was wrong last night.   I cannot reproduce the problem. 
Closing.
